core/market/decentralized/src/protocol/negotiation/errors.rs

```diff
@@ -22,6 +45,12 @@ impl From<ya_service_bus::error::Error> for ProposalError {
     }
 }
 
+impl From<ya_service_bus::error::Error> for CounterProposalError {
```

Comment:
As far as I remember `ya_service_bus::error::Error` isn't serializable, so it can't be embedded, because `CounterProposalError` will be sent through GSB.
So I must dispatch it into string